### ISSUE TITLE
{label_settings_en}

### ISSUE THREAD

**Issue by pdimu** [PranshulGG/WeatherMaster#29]
Hi! When scrolling down through Settings, `{label_settings_en}` is appearing next to the back arrow
Screenshot:
![Screenshot_2024-09-10-15-26-36-817-edit_com.example.weathermaster.jpg](https://github.com/user-attachments/assets/599f6684-14ec-4663-a90e-ca7dbb503ffe)

**Comment by PranshulGG** [comment]
Yeah, I added translation support to the app, so maybe I forgot to include the word 'settings.

**Comment by pdimu** [comment]
Oh, and WeatherMaster told me there's a new update when the latest version is 1.5.8 (my current version)

**Comment by pdimu** [comment]
Very clever
